classwork/classwork3.2/include/Color.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <tuple>
+
+class Color {
+public:
+    Color(int r, int g, int b);
+
+    std::tuple<int, int, int> getRGB() const;
+
+    Color magicColor() const;
```

Comment:
minor: can be [[nodiscard]]. Also we can use better name like generateMagicColor or something like that to describe that some calculation are performed not just getter call